lib/Basics/files.cpp

```diff
@@ -1465,9 +1465,9 @@ int TRI_VerifyLockFile(char const* filename) {
   // file was not yet locked; could be locked
   if (canLock == 0) {
     // lock.l_type = F_UNLCK;
-    res = fcntl(fd, F_GETLK, &lock);
+    int res2 = fcntl(fd, F_GETLK, &lock);
 
-    if (res != TRI_ERROR_NO_ERROR) {
+    if (res2 != 0) {
```

Comment:
`res2` is not really needed here. Maybe make this more readable by simplifying it to
```
if (0 != fcntl(fd, F_GETLK, &lock)) {
  TRI_set_errno(TRI_ERROR_SYS_ERROR);
  ...
```

Reply:
Done.